roles/lmod/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+#
+# ansible role to install lmod:
+#
+---
+- name: "install packages"
+  become: yes
+  action: apt name=lmod state=latest
+  when: ansible_os_family == "Debian"
+
+- name: "install packages"
+  become: yes
+  action: yum name=lmod state=latest
+  when: ansible_os_family == "RedHat"
+
+- name: "mkdir software path"
+  file: path={{ eb_software_path }} state=directory
+
+- name: "mkdir module path"
+  file: path={{ eb_modulepath }} state=directory 
```

Comment:
Should use the extended format to be consistent with our other roles and the other tasks in this role